MagicSpell/InstrPageViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  InstrPageViewController.swift
+//  MagicSpell
+//
+//  Created by Lingyi Li on 8/31/16.
+//  Copyright © 2016 Georgia Tech. All rights reserved.
+//
+
+import UIKit
+
+class InstrPageViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    var tableViewForInstruction: UITableView!
+    var cellForInstruction : InstrPageTableViewCell!
+    var labelForTitle : UILabel!
+    var labelForSubtitle : UILabel!
+    var btnForAlreadyDone : UIButton!
+    
+    var instructionSet = ["1.  Open your device settings", "2.  General", "3.  Keyboard", "4.  Keyboards", "5.  Add new keyboard", "6.  MagicSpell keyboard", "7.  Tap MagicSpell", "8.  Allow Full access"]
```

Comment:
s/settings/Settings
s/Add new keyboard/Add New Keyboard
s/MagicSpell keyboard/MagicSpell
s/Tap MagicSpell/Tap "MagicSpellKeyboard -- MagicSpell"
Remove "Allow Full Access" -  we don't need special permissions for the keyboard.